2023-06-12 16:31:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "CREATE TABLE USERS (
  ID INT PRIMARY KEY AUTO_INCREMENT,
  MAIL_ADDRESS VARCHAR(50) UNIQUE KEY[*] NOT NULL,
  USERPW VARCHAR(20) NOT NULL
)"; 期待されるステートメント "NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"
Syntax error in SQL statement "CREATE TABLE USERS (
  ID INT PRIMARY KEY AUTO_INCREMENT,
  MAIL_ADDRESS VARCHAR(50) UNIQUE KEY[*] NOT NULL,
  USERPW VARCHAR(20) NOT NULL
)"; expected "NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"; SQL statement:
CREATE TABLE users (
  ID INT PRIMARY KEY AUTO_INCREMENT,
  MAIL_ADDRESS VARCHAR(50) UNIQUE KEY NOT NULL,
  USERPW VARCHAR(20) NOT NULL
) [42001-200]
2023-06-12 16:33:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "CREATE TABLE USERS (
  ID INT AUTO_INCREMENT PRIMARY KEY ,
  MAIL_ADDRESS VARCHAR(50) UNIQUE KEY[*] NOT NULL,
  USERPW VARCHAR(20) NOT NULL
)"; 期待されるステートメント "NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"
Syntax error in SQL statement "CREATE TABLE USERS (
  ID INT AUTO_INCREMENT PRIMARY KEY ,
  MAIL_ADDRESS VARCHAR(50) UNIQUE KEY[*] NOT NULL,
  USERPW VARCHAR(20) NOT NULL
)"; expected "NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"; SQL statement:
CREATE TABLE users (
  ID INT AUTO_INCREMENT PRIMARY KEY ,
  MAIL_ADDRESS VARCHAR(50) UNIQUE KEY NOT NULL,
  USERPW VARCHAR(20) NOT NULL
) [42001-200]
2023-06-12 16:33:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "CREATE TABLE USERS (
  ID INT AUTO_INCREMENT PRIMARY KEY ,
  MAIL_ADDRESS VARCHAR(50)  NOT NULL UNIQUE KEY[*],
  USERPW VARCHAR(20) NOT NULL
)"; 期待されるステートメント "NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"
Syntax error in SQL statement "CREATE TABLE USERS (
  ID INT AUTO_INCREMENT PRIMARY KEY ,
  MAIL_ADDRESS VARCHAR(50)  NOT NULL UNIQUE KEY[*],
  USERPW VARCHAR(20) NOT NULL
)"; expected "NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"; SQL statement:
CREATE TABLE users (
  ID INT AUTO_INCREMENT PRIMARY KEY ,
  MAIL_ADDRESS VARCHAR(50)  NOT NULL UNIQUE KEY,
  USERPW VARCHAR(20) NOT NULL
) [42001-200]
2023-06-12 16:42:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "CREATE TABLE USERS (
  ID INT PRIMARY KEY AUTO_INCREMENT,
  MAIL_ADDRESS VARCHAR(50)  NOT NULL UNIQUE KEY[*],
  USERPW VARCHAR(20) NOT NULL
)"; 期待されるステートメント "NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"
Syntax error in SQL statement "CREATE TABLE USERS (
  ID INT PRIMARY KEY AUTO_INCREMENT,
  MAIL_ADDRESS VARCHAR(50)  NOT NULL UNIQUE KEY[*],
  USERPW VARCHAR(20) NOT NULL
)"; expected "NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"; SQL statement:
CREATE TABLE users (
  ID INT PRIMARY KEY AUTO_INCREMENT,
  MAIL_ADDRESS VARCHAR(50)  NOT NULL UNIQUE KEY,
  USERPW VARCHAR(20) NOT NULL
) [42001-200]
2023-06-12 16:43:29 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "CREATE TABLE USERS (
  ID INT PRIMARY KEY AUTO_INCREMENT,
  MAIL_ADDRESS VARCHAR(50) UNIQUE KEY[*] NOT NULL,
  USERPW VARCHAR(20) NOT NULL
)"; 期待されるステートメント "NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"
Syntax error in SQL statement "CREATE TABLE USERS (
  ID INT PRIMARY KEY AUTO_INCREMENT,
  MAIL_ADDRESS VARCHAR(50) UNIQUE KEY[*] NOT NULL,
  USERPW VARCHAR(20) NOT NULL
)"; expected "NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"; SQL statement:
CREATE TABLE users (
  ID INT PRIMARY KEY AUTO_INCREMENT,
  MAIL_ADDRESS VARCHAR(50) UNIQUE KEY NOT NULL,
  USERPW VARCHAR(20) NOT NULL
) [42001-200]
2023-06-12 16:43:46 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "CREATE TABLE USERS (
  ID INT PRIMARY KEY AUTO_INCREMENT,
  MAIL_ADDRESS VARCHAR(50) NOT NULL  UNIQUE KEY[*] ,
  USERPW VARCHAR(20) NOT NULL
)"; 期待されるステートメント "NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"
Syntax error in SQL statement "CREATE TABLE USERS (
  ID INT PRIMARY KEY AUTO_INCREMENT,
  MAIL_ADDRESS VARCHAR(50) NOT NULL  UNIQUE KEY[*] ,
  USERPW VARCHAR(20) NOT NULL
)"; expected "NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"; SQL statement:
CREATE TABLE users (
  ID INT PRIMARY KEY AUTO_INCREMENT,
  MAIL_ADDRESS VARCHAR(50) NOT NULL  UNIQUE KEY ,
  USERPW VARCHAR(20) NOT NULL
) [42001-200]
2023-06-13 09:54:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "CREATE TABLE USERS (
  ID INT PRIMARY KEY AUTO_INCREMENT,
  MAIL_ADDRESS VARCHAR(50) UNIQUE KEY[*] NOT NULL,
  USERPW VARCHAR(20) NOT NULL
)"; 期待されるステートメント "NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"
Syntax error in SQL statement "CREATE TABLE USERS (
  ID INT PRIMARY KEY AUTO_INCREMENT,
  MAIL_ADDRESS VARCHAR(50) UNIQUE KEY[*] NOT NULL,
  USERPW VARCHAR(20) NOT NULL
)"; expected "NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"; SQL statement:
CREATE TABLE users (
  ID INT PRIMARY KEY AUTO_INCREMENT,
  MAIL_ADDRESS VARCHAR(50) UNIQUE KEY NOT NULL,
  USERPW VARCHAR(20) NOT NULL
) [42001-200]
2023-06-13 16:56:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列 "USEPW" が見つかりません
Column "USEPW" not found; SQL statement:
INSERT INTO users(ID, MAIL_ADDRESS, USEPW) VALUES (0001,'mail1@company1.jp','a1a1') [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:703)
	at org.h2.command.Parser.parseColumn(Parser.java:1450)
	at org.h2.command.Parser.parseColumnList(Parser.java:1435)
	at org.h2.command.Parser.parseInsertGivenTable(Parser.java:1804)
	at org.h2.command.Parser.parseInsert(Parser.java:1749)
	at org.h2.command.Parser.parsePrepared(Parser.java:954)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2023-06-13 16:56:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列 "USEPW" が見つかりません
Column "USEPW" not found; SQL statement:
INSERT INTO users(ID, MAIL_ADDRESS, USEPW) VALUES (0002,'mail2@company2.jp','b2b2') [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:703)
	at org.h2.command.Parser.parseColumn(Parser.java:1450)
	at org.h2.command.Parser.parseColumnList(Parser.java:1435)
	at org.h2.command.Parser.parseInsertGivenTable(Parser.java:1804)
	at org.h2.command.Parser.parseInsert(Parser.java:1749)
	at org.h2.command.Parser.parsePrepared(Parser.java:954)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2023-06-13 16:56:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列 "USEPW" が見つかりません
Column "USEPW" not found; SQL statement:
INSERT INTO users(ID, MAIL_ADDRESS, USEPW) VALUES (0003,'mail3@company3.jp','c3c3') [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:703)
	at org.h2.command.Parser.parseColumn(Parser.java:1450)
	at org.h2.command.Parser.parseColumnList(Parser.java:1435)
	at org.h2.command.Parser.parseInsertGivenTable(Parser.java:1804)
	at org.h2.command.Parser.parseInsert(Parser.java:1749)
	at org.h2.command.Parser.parsePrepared(Parser.java:954)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2023-06-13 16:56:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列 "USEPW" が見つかりません
Column "USEPW" not found; SQL statement:
INSERT INTO users(ID, MAIL_ADDRESS, USEPW) VALUES (0004,'mail4@company4.jp','d4d4') [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:703)
	at org.h2.command.Parser.parseColumn(Parser.java:1450)
	at org.h2.command.Parser.parseColumnList(Parser.java:1435)
	at org.h2.command.Parser.parseInsertGivenTable(Parser.java:1804)
	at org.h2.command.Parser.parseInsert(Parser.java:1749)
	at org.h2.command.Parser.parsePrepared(Parser.java:954)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2023-06-13 16:56:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列 "USEPW" が見つかりません
Column "USEPW" not found; SQL statement:
INSERT INTO users(ID, MAIL_ADDRESS, USEPW) VALUES (0005,'mail5@company5.jp','e5e5') [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:703)
	at org.h2.command.Parser.parseColumn(Parser.java:1450)
	at org.h2.command.Parser.parseColumnList(Parser.java:1435)
	at org.h2.command.Parser.parseInsertGivenTable(Parser.java:1804)
	at org.h2.command.Parser.parseInsert(Parser.java:1749)
	at org.h2.command.Parser.parsePrepared(Parser.java:954)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2023-06-13 16:57:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列 "PASSWORD" が見つかりません
Column "PASSWORD" not found; SQL statement:
INSERT INTO users(ID, MAIL_ADDRESS, PASSWORD) VALUES (0001,'mail1@company1.jp','a1a1') [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:703)
	at org.h2.command.Parser.parseColumn(Parser.java:1450)
	at org.h2.command.Parser.parseColumnList(Parser.java:1435)
	at org.h2.command.Parser.parseInsertGivenTable(Parser.java:1804)
	at org.h2.command.Parser.parseInsert(Parser.java:1749)
	at org.h2.command.Parser.parsePrepared(Parser.java:954)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2023-06-13 16:57:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列 "PASSWORD" が見つかりません
Column "PASSWORD" not found; SQL statement:
INSERT INTO users(ID, MAIL_ADDRESS, PASSWORD) VALUES (0002,'mail2@company2.jp','b2b2') [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:703)
	at org.h2.command.Parser.parseColumn(Parser.java:1450)
	at org.h2.command.Parser.parseColumnList(Parser.java:1435)
	at org.h2.command.Parser.parseInsertGivenTable(Parser.java:1804)
	at org.h2.command.Parser.parseInsert(Parser.java:1749)
	at org.h2.command.Parser.parsePrepared(Parser.java:954)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2023-06-13 16:57:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列 "PASSWORD" が見つかりません
Column "PASSWORD" not found; SQL statement:
INSERT INTO users(ID, MAIL_ADDRESS, PASSWORD) VALUES (0003,'mail3@company3.jp','c3c3') [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:703)
	at org.h2.command.Parser.parseColumn(Parser.java:1450)
	at org.h2.command.Parser.parseColumnList(Parser.java:1435)
	at org.h2.command.Parser.parseInsertGivenTable(Parser.java:1804)
	at org.h2.command.Parser.parseInsert(Parser.java:1749)
	at org.h2.command.Parser.parsePrepared(Parser.java:954)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2023-06-13 16:57:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列 "PASSWORD" が見つかりません
Column "PASSWORD" not found; SQL statement:
INSERT INTO users(ID, MAIL_ADDRESS, PASSWORD) VALUES (0004,'mail4@company4.jp','d4d4') [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:703)
	at org.h2.command.Parser.parseColumn(Parser.java:1450)
	at org.h2.command.Parser.parseColumnList(Parser.java:1435)
	at org.h2.command.Parser.parseInsertGivenTable(Parser.java:1804)
	at org.h2.command.Parser.parseInsert(Parser.java:1749)
	at org.h2.command.Parser.parsePrepared(Parser.java:954)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2023-06-13 16:57:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列 "PASSWORD" が見つかりません
Column "PASSWORD" not found; SQL statement:
INSERT INTO users(ID, MAIL_ADDRESS, PASSWORD) VALUES (0005,'mail5@company5.jp','e5e5') [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:703)
	at org.h2.command.Parser.parseColumn(Parser.java:1450)
	at org.h2.command.Parser.parseColumnList(Parser.java:1435)
	at org.h2.command.Parser.parseInsertGivenTable(Parser.java:1804)
	at org.h2.command.Parser.parseInsert(Parser.java:1749)
	at org.h2.command.Parser.parsePrepared(Parser.java:954)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2023-06-13 17:00:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: "DATE" 定数 "2023/01/01" を解析できません
Cannot parse "DATE" constant "2023/01/01"; SQL statement:
INSERT INTO posts (ID, USERS_ID, DATE, PHOTO, RESTAURANT, WALK, SERVE, PRICE, GENRE, TEXT, POINT) 
VALUES (1,0001, '2023/01/01', '写真1','店名1', 1, 1, 1, 1, 'テキスト1', 100) [22007-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.value.ValueDate.parse(ValueDate.java:74)
	at org.h2.value.Value.convertToDate(Value.java:1093)
	at org.h2.value.Value.convertTo(Value.java:816)
	at org.h2.value.Value.convertTo(Value.java:772)
	at org.h2.value.TypeInfo.cast(TypeInfo.java:515)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:378)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:845)
	at org.h2.command.dml.Insert.insertRows(Insert.java:187)
	at org.h2.command.dml.Insert.update(Insert.java:151)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.NumberFormatException: For input string: "2023/0"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.base/java.lang.Integer.parseInt(Integer.java:652)
	at java.base/java.lang.Integer.parseInt(Integer.java:770)
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:169)
	at org.h2.value.ValueDate.parse(ValueDate.java:72)
	... 19 more
2023-06-13 17:00:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: "DATE" 定数 "2023/02/02" を解析できません
Cannot parse "DATE" constant "2023/02/02"; SQL statement:
INSERT INTO posts (ID, USERS_ID, DATE, PHOTO, RESTAURANT, WALK, SERVE, PRICE, GENRE, TEXT, POINT) 
VALUES (2,0002, '2023/02/02', '写真2','店名2', 2, 2, 2, 2, 'テキスト2', 200) [22007-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.value.ValueDate.parse(ValueDate.java:74)
	at org.h2.value.Value.convertToDate(Value.java:1093)
	at org.h2.value.Value.convertTo(Value.java:816)
	at org.h2.value.Value.convertTo(Value.java:772)
	at org.h2.value.TypeInfo.cast(TypeInfo.java:515)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:378)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:845)
	at org.h2.command.dml.Insert.insertRows(Insert.java:187)
	at org.h2.command.dml.Insert.update(Insert.java:151)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.NumberFormatException: For input string: "2023/0"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.base/java.lang.Integer.parseInt(Integer.java:652)
	at java.base/java.lang.Integer.parseInt(Integer.java:770)
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:169)
	at org.h2.value.ValueDate.parse(ValueDate.java:72)
	... 19 more
2023-06-13 17:00:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: "DATE" 定数 "2023/03/03" を解析できません
Cannot parse "DATE" constant "2023/03/03"; SQL statement:
INSERT INTO posts (ID, USERS_ID, DATE, PHOTO, RESTAURANT, WALK, SERVE, PRICE, GENRE, TEXT, POINT) 
VALUES (3,0003, '2023/03/03', '写真3','店名3', 3, 3, 3, 3, 'テキスト3', 300) [22007-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.value.ValueDate.parse(ValueDate.java:74)
	at org.h2.value.Value.convertToDate(Value.java:1093)
	at org.h2.value.Value.convertTo(Value.java:816)
	at org.h2.value.Value.convertTo(Value.java:772)
	at org.h2.value.TypeInfo.cast(TypeInfo.java:515)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:378)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:845)
	at org.h2.command.dml.Insert.insertRows(Insert.java:187)
	at org.h2.command.dml.Insert.update(Insert.java:151)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.NumberFormatException: For input string: "2023/0"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.base/java.lang.Integer.parseInt(Integer.java:652)
	at java.base/java.lang.Integer.parseInt(Integer.java:770)
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:169)
	at org.h2.value.ValueDate.parse(ValueDate.java:72)
	... 19 more
2023-06-13 17:00:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: "DATE" 定数 "126" を解析できません
Cannot parse "DATE" constant "126"; SQL statement:
INSERT INTO posts (ID, USERS_ID, DATE, PHOTO, RESTAURANT, WALK, SERVE, PRICE, GENRE, TEXT, POINT) 
VALUES (4,0004, 2023/04/04, '写真4','店名4', 4, 4, 4, 4, 'テキスト4', 400) [22007-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.value.ValueDate.parse(ValueDate.java:74)
	at org.h2.value.Value.convertToDate(Value.java:1093)
	at org.h2.value.Value.convertTo(Value.java:816)
	at org.h2.value.Value.convertTo(Value.java:772)
	at org.h2.value.TypeInfo.cast(TypeInfo.java:515)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:378)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:845)
	at org.h2.command.dml.Insert.insertRows(Insert.java:187)
	at org.h2.command.dml.Insert.update(Insert.java:151)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.IllegalArgumentException: 126
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:166)
	at org.h2.value.ValueDate.parse(ValueDate.java:72)
	... 19 more
2023-06-13 17:00:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: "DATE" 定数 "2023/05/05" を解析できません
Cannot parse "DATE" constant "2023/05/05"; SQL statement:
INSERT INTO posts (ID, USERS_ID, DATE, PHOTO, RESTAURANT, WALK, SERVE, PRICE, GENRE, TEXT, POINT) 
VALUES (5,0005, '2023/05/05', '写真5','店名5', 5, 5, 5, 5, 'テキスト5', 500) [22007-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.value.ValueDate.parse(ValueDate.java:74)
	at org.h2.value.Value.convertToDate(Value.java:1093)
	at org.h2.value.Value.convertTo(Value.java:816)
	at org.h2.value.Value.convertTo(Value.java:772)
	at org.h2.value.TypeInfo.cast(TypeInfo.java:515)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:378)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:845)
	at org.h2.command.dml.Insert.insertRows(Insert.java:187)
	at org.h2.command.dml.Insert.update(Insert.java:151)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.NumberFormatException: For input string: "2023/0"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.base/java.lang.Integer.parseInt(Integer.java:652)
	at java.base/java.lang.Integer.parseInt(Integer.java:770)
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:169)
	at org.h2.value.ValueDate.parse(ValueDate.java:72)
	... 19 more
2023-06-13 17:01:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: "DATE" 定数 "2015" を解析できません
Cannot parse "DATE" constant "2015"; SQL statement:
INSERT INTO posts (ID, USERS_ID, DATE, PHOTO, RESTAURANT, WALK, SERVE, PRICE, GENRE, TEXT, POINT) 
VALUES (4,0004, 2023-04-04, '写真4','店名4', 4, 4, 4, 4, 'テキスト4', 400) [22007-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.value.ValueDate.parse(ValueDate.java:74)
	at org.h2.value.Value.convertToDate(Value.java:1093)
	at org.h2.value.Value.convertTo(Value.java:816)
	at org.h2.value.Value.convertTo(Value.java:772)
	at org.h2.value.TypeInfo.cast(TypeInfo.java:515)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:378)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:845)
	at org.h2.command.dml.Insert.insertRows(Insert.java:187)
	at org.h2.command.dml.Insert.update(Insert.java:151)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.IllegalArgumentException: 2015
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:166)
	at org.h2.value.ValueDate.parse(ValueDate.java:72)
	... 19 more
2023-06-13 17:03:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列 "POST_ID" が見つかりません
Column "POST_ID" not found; SQL statement:
INSERT INTO COMMENTS(ID, USERS_ID, POST_ID, DATE, POINT, TEXT) VALUES(1,0001,1,'2023-01-01',1,'コメント1') [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:703)
	at org.h2.command.Parser.parseColumn(Parser.java:1450)
	at org.h2.command.Parser.parseColumnList(Parser.java:1435)
	at org.h2.command.Parser.parseInsertGivenTable(Parser.java:1804)
	at org.h2.command.Parser.parseInsert(Parser.java:1749)
	at org.h2.command.Parser.parsePrepared(Parser.java:954)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2023-06-13 17:03:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列 "POST_ID" が見つかりません
Column "POST_ID" not found; SQL statement:
INSERT INTO COMMENTS(ID, USERS_ID, POST_ID, DATE, POINT, TEXT) VALUES(2,0002,2,'2023-02-02',2,'コメント2') [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:703)
	at org.h2.command.Parser.parseColumn(Parser.java:1450)
	at org.h2.command.Parser.parseColumnList(Parser.java:1435)
	at org.h2.command.Parser.parseInsertGivenTable(Parser.java:1804)
	at org.h2.command.Parser.parseInsert(Parser.java:1749)
	at org.h2.command.Parser.parsePrepared(Parser.java:954)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2023-06-13 17:03:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列 "POST_ID" が見つかりません
Column "POST_ID" not found; SQL statement:
INSERT INTO COMMENTS(ID, USERS_ID, POST_ID, DATE, POINT, TEXT) VALUES(3,0003,3,'2023-03-03',3,'コメント3') [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:703)
	at org.h2.command.Parser.parseColumn(Parser.java:1450)
	at org.h2.command.Parser.parseColumnList(Parser.java:1435)
	at org.h2.command.Parser.parseInsertGivenTable(Parser.java:1804)
	at org.h2.command.Parser.parseInsert(Parser.java:1749)
	at org.h2.command.Parser.parsePrepared(Parser.java:954)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2023-06-13 17:03:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列 "POST_ID" が見つかりません
Column "POST_ID" not found; SQL statement:
INSERT INTO COMMENTS(ID, USERS_ID, POST_ID, DATE, POINT, TEXT) VALUES(4,0004,4,'2023-04-04',4,'コメント4') [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:703)
	at org.h2.command.Parser.parseColumn(Parser.java:1450)
	at org.h2.command.Parser.parseColumnList(Parser.java:1435)
	at org.h2.command.Parser.parseInsertGivenTable(Parser.java:1804)
	at org.h2.command.Parser.parseInsert(Parser.java:1749)
	at org.h2.command.Parser.parsePrepared(Parser.java:954)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2023-06-13 17:03:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列 "POST_ID" が見つかりません
Column "POST_ID" not found; SQL statement:
INSERT INTO COMMENTS(ID, USERS_ID, POST_ID, DATE, POINT, TEXT) VALUES(5,0005,5,'2023-05-05',5,'コメント5') [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:703)
	at org.h2.command.Parser.parseColumn(Parser.java:1450)
	at org.h2.command.Parser.parseColumnList(Parser.java:1435)
	at org.h2.command.Parser.parseInsertGivenTable(Parser.java:1804)
	at org.h2.command.Parser.parseInsert(Parser.java:1749)
	at org.h2.command.Parser.parsePrepared(Parser.java:954)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
